src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -442,19 +449,20 @@ export class HuiCardPicker extends LitElement {
       }
     }
 
+    // prevent tabbing to card
+    if (element) {
+      element.tabIndex = -1;
+    }
+
     return html`
-      <div class="card">
+      <div class="card" tabindex="0">
         <div
           class="overlay"
           @click=${this._cardPicked}
           .config=${cardConfig}
         ></div>
         <div class="card-header">
-          ${customCard
-            ? `${this.hass!.localize(
-                "ui.panel.lovelace.editor.cardpicker.custom_card"
-              )}: ${customCard.name || customCard.type}`
-            : name}
```

Comment:
@jpbede why do we remove the custom label?
For integrations we have a own icon that it's a custom one, but here you would not see that it's custom when we remove the label.